app/services/sessions/manager.rb

```diff
@@ -41,6 +44,12 @@ def check_authorisation!(session_user)
       fail(MissingAccessLevel)
     end
 
+    def encrypt_token(token)
+      secret_key = Rails.application.secret_key_base.byteslice(0, 32)
```

Comment:
I feel like we should set this as a a separate environment variable called `DFE_SIGN_IN_ID_TOKEN_KEY` or similar, which would make rotating easier should we have to.

Reply:
Yeah. Was just thinking if for some reason we need to rotate it, it's easier than doing the Rails creds too